azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/repository/MessageRepository.kt

```diff
@@ -20,15 +20,15 @@ private val emptyMessage = MessageInfoModel(
 // - pageRetrieved
 // - messageEdited
 // - messageDeleted
-internal interface MessageRepositoryMiddlewareInterface {
+internal interface MessageRepositoryMiddlewareInterface : List<MessageInfoModel> {
```

Comment:
List should stay below on MessageRepository, but not on the MiddlewareInterface

Reply:
updated.